frontend/src/metabase/query_builder/components/view/QuestionRowCount/QuestionRowCount.tsx

```diff
@@ -78,6 +77,7 @@ function QuestionRowCount({
   className,
   onChangeLimit,
 }: QuestionRowCountProps) {
+  const [opened, setOpened] = useState(false);
```

Comment:
Let's use the `useDisclosure` here, please.
I think this is our preferred way of dealing with popover and modal states.